man/bootup.xml

```diff
@@ -23,17 +23,25 @@
   <refsect1>
     <title>Description</title>
 
-    <para>A number of different components are involved in the system
-    boot. Immediately after power-up, the system BIOS will do minimal
-    hardware initialization, and hand control over to a boot loader
-    stored on a persistent storage device. This boot loader will then
-    invoke an OS kernel from disk (or the network). In the Linux case,
-    this kernel (optionally) extracts and executes an initial RAM disk
-    image (initrd), such as generated by
+    <para>A number of different components are involved in the boot of a Linux system. Immediately after
+    power-up, the system BIOS will do minimal hardware initialization, and hand control over to a boot loader
+    (e.g. <ulink url="https://www.gnu.org/software/grub/">GRUB</ulink> or
+    <citerefentry><refentrytitle>systemd-boot</refentrytitle><manvolnum>7</manvolnum></citerefentry>)
+    stored on a persistent storage device. This boot loader will then invoke an OS kernel from disk (or the
+    network). On systems using EFI or other types of firmware, this firmware may also load the kernel
+    directly.</para>
+
+    <para>The kernel (optionally) mounts an in-memory file system, such as generated by
     <citerefentry project='die-net'><refentrytitle>dracut</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
-    which looks for the root file system (possibly using
-    <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>
-    for this). After the root file system is found and mounted, the
+    which looks for the root file system. Initramfs — an lightweight read-only in-memory file system based on
+    tmpfs is usually used for this, but in the past normal file systems using an in-memory block device
+    (ramdisk) were used, and the name "initrd" is still used to describe both mechanisms. It's the boot loader
+    or the firmware that loads both the kernel and initrd/initramfs images into memory, but the kernel which
+    interprets it as a file system.
+    <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry> may be used
+    to manage services in the initrd, similarly to the real system.</para>
```

Comment:
so, strictly speaking initrds are not a file system but cpio archives which are unpacked by the kernel early on into a file system. When we explain the concept we should probably get this right.

Reply:
Yikes, I misunderstood how the initramfs works. I updated the description.